x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -247,36 +247,47 @@ const getProductTypes = (
 export const cli = () => {
   run(
     async (context) => {
+      const environment = process.env.ENVIRONMENT ? `${process.env.ENVIRONMENT}` : 'qa';
+
+      const PROXY_URL = process.env.PROXY_URL ? process.env.PROXY_URL : undefined;
+      const PROXY_SECRET = process.env.PROXY_SECRET || undefined;
```

Comment:
```suggestion
      const PROXY_SECRET = process.env.PROXY_SECRET;
```

`process.env.PROXY_SECRET`'s type is `string | undefined`. Do we expect something else here?